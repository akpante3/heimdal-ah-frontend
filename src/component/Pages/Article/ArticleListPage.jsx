import React from 'react';
import ArticleList from '../../../container/Article/Article';
import '../../ui/cards/ArticleCard/articleCard.scss';

/**
 * Article List page
 */
export default function ArticleListPage() {
  return (
    <div className='mb-5'>
      <h2 className='mx-auto text-center'> All Stories</h2>
      <ArticleList />

    </div>
  );
}
