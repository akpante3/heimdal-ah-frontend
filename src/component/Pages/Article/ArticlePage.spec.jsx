/* eslint-disable no-undef */
import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import ArticleListPage from './ArticleListPage';

describe('Mount Article page', () => {
  it('should render the article page', () => {
    const tree = shallow(<ArticleListPage />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
