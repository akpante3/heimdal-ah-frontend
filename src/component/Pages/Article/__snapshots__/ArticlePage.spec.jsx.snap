// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mount Article page should render the article page 1`] = `
<div
  className="mb-5 p-5"
>
  <h2
    className="mx-auto text-center text-muted"
  >
     All Stories
  </h2>
  <Connect(Article) />
</div>
`;
